frontend/next.js/BUILD.bazel

```diff
@@ -80,3 +81,66 @@ build_test(
         # ":next_export",
     ],
 )
+
+platform(
+    name = "linux_amd64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+platform(
+    name = "linux_arm64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+next_bin.next_binary(
```

Comment:
Instead of a `js_run_devserver` binary which is for local development and won't work in a container, what you want for the image is just a `js_binary` which is what `next_bin.next_binary` is under the hood